packages/core/test/integration/query-interface.test.js

```diff
@@ -712,6 +712,24 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         constraints = await this.queryInterface.showConstraint('posts');
         constraints = constraints.map(constraint => constraint.constraintName);
         expect(constraints).to.not.include('posts_username_users_fk');
+
+        await this.queryInterface.addConstraint('posts', {
```

Comment:
could you add a multi-fields constraint test to `integration/query-interface/add-constraint.test.ts` instead? This way we'll both test that this actually works, and that the typings are correct :)

Reply:
Done

Reply:
I think you missed the part about moving it to a TS file, could you move it to `integration/query-interface/add-constraint.test.ts`? :)
Otherwise the typings won't be tested

Reply:
Oops, yeah, totally misread that - will move it into a new file.